test/e2e/tests/test_layer_version.py

```diff
@@ -79,6 +79,19 @@ def test_smoke(self, lambda_client):
         # Check layer version exists
         assert lambda_validator.layer_version_exists(resource_name, version_number)
 
+        logging.info(cr['spec']['description'])
+        # Update cr
+        cr['spec']['description'] = 'new description'
```

Comment:
@Vandita2020 Instead of passing in an updated `cr` object, try just passing in the things you want to update.

Follow this example which shows you how to do this:

https://github.com/aws-controllers-k8s/iam-controller/blob/f528ed16a631d2b57f4fd06eb1abc1dac08ea099/test/e2e/tests/test_role.py#L99-L105

(I agree it's a bit weird! :) )

Reply:
@jaypipes Thanks so much for sharing this, I was really looking for another way to update the fields.